avocado-plugins-vt.spec

```diff
@@ -21,7 +21,7 @@ Requires: python-imaging
 %if 0%{?el6}
 Requires: gstreamer-python, gstreamer-plugins-good, python-simplejson
 %else
-Requires: pygobject2, gstreamer1-plugins-good
+Requires: pygobject2, gstreamer1-plugins-good, python2-simplejson
```

Comment:
And I've tested that such a test produces a package that brings in the right dep:

```
Dependencies Resolved

==================================================================================================================================================
 Package                        Arch               Version                         Repository                                                Size
==================================================================================================================================================
Installing:
 avocado-plugins-vt             noarch             47.0-0.el7.centos               /avocado-plugins-vt-47.0-0.el7.centos.noarch              15 M
Installing for dependencies:
 python-netaddr                 noarch             0.7.5-7.el7                     base                                                     983 k
 python-netifaces               x86_64             0.10.4-3.el7                    base                                                      17 k
 python2-simplejson             x86_64             3.10.0-1.el7                    epel                                                     188 k

Transaction Summary
==================================================================================================================================================
Install  1 Package (+3 Dependent packages)
```